### PR TITLE
Robotics page copy update

### PR DESCRIPTION
**Pull request by carkod**
## Done

Copy updates for robotics page

## QA

- Go to https://ubuntu-com-11582.demos.haus/robotics and check [copy docs](https://docs.google.com/document/d/1JNFk2C0shH3DapKuJM8X0QfAiD1RzHvXoydn8P9mEMg/edit#)
- Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5114

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/robotics/index.html [hunk 1]
```diff
@@ -13,31 +13,49 @@
   {% include "shared/_rt-kernel-beta-banner.html" %}
   <div class="row u-equal-height">
     <div class="col-8">
-      <h1>Build security into your robot's future with Ubuntu</h1>
-      <h4>Tech visionaries choose Ubuntu to build ultra-secure and software-defined robots</h4>
+      <h1>Powering robotics in every industry</h1>
+      <h4>Develop, deploy and maintain your fleet of robots with Ubuntu.</h4>
       <p>Ubuntu is the most popular Linux distribution for large embedded systems. As autonomous robots mature, innovative tech companies turn to Ubuntu to realise their vision of a robotic future. We are your partner in innovation.</p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:18]: This content is not in the copy doc.

File: templates/robotics/index.html [hunk 2]
```diff
@@ -13,31 +13,49 @@
   {% include "shared/_rt-kernel-beta-banner.html" %}
```
Comment (anthonydillon) [inline templates/robotics/index.html:13]: Do we still want this?

File: templates/robotics/index.html [hunk 3]
```diff
@@ -13,31 +13,49 @@
...
   </div>
 </section>
 
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Ubuntu is all your robot needs</h2>
+    <h4>We provide you with the essential infrastructure to easily, securely, and reliably deploy your robots</h4>
+    <p>Our open source tools help you:</p>
+    <ul class="p-list">
+      <li class="p-list__item is-ticked">Keep your robots up-to-date efficiently and cost-effectively with over the air and low bandwidth updates with failure rollback.</li>
```
Comment (anthonydillon) [inline templates/robotics/index.html:43]: Should be "over-the-air"

File: templates/robotics/index.html [hunk 4]
```diff
@@ -78,6 +96,33 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
             ) | safe
           }}
         </div>
+
+        <div class="p-logo-section__item">
```
Comment (anthonydillon) [inline templates/robotics/index.html:100]: These logos should be produced by design @lyubomir-popov

File: templates/robotics/index.html [hunk 5]
```diff
@@ -133,32 +178,56 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
         </div>
       </div>
     </div>
-    <p class="u-align--center"><a href="https://partners.ubuntu.com/find-a-partner?technology=robotics">See all of our robotics partners</a></p>
+    <p class="u-align--center"><a href="https://partners.ubuntu.com/find-a-partner?technology=robotics">See all of our robotics partners&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:181]:
```suggestion
    <p class="u-align--center"><a href="https://canonical.com/partners/find-a-partner?filters=robotics">See all of our robotics partners&nbsp;&rsaquo;</a></p>
```

File: templates/robotics/index.html [hunk 6]
```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
...
+      <h4>Comply with cybersecurity requirements with in-built security for your device</h4>
+      <p>With Ubuntu Core, in-built security means:</p>
+      <ul>
+        <li><a href="/core/features/secure-boot">Secure boot</a></li>
+        <li><a href="/engage/iot-disk-encryption">Full disk encryption</a></li>
+        <li><a href="https://snapcraft.io/docs/assertions">Signed software packages</a></li>
+        <li><a href="/core/features/recovery">Device recovery</a></li>
+        <li>10-year security update commitment</li>
+      </ul>
+      <p>Snap confinement keeps your apps separated and controls access to hardware.</p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:261]: "[Snap confinement](https://snapcraft.io/docs/snap-confinement)" should be a link

File: templates/robotics/index.html [hunk 7]
```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
...
+      <p>Curate your IoT App Store from global suppliers, with constant application security scanning and updates.</p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:262]: "[IoT App Store](https://ubuntu.com/internet-of-things/appstore)" should be a link

File: templates/robotics/index.html [hunk 8]
```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
...
 </section>
 
 <section class="p-strip--light">
   <div class="u-fixed-width">
-    <h2>Catalyses new business models </h2>
-    <p>Improve the return on investment for your robotic assets.</p>
-    <a href="https://www.brighttalk.com/webcast/6793/290087">Watch the app store for Linux webinar</a>
+    <h2>Enterprise services for robotics companies</h2>
+    <h4>Deploy robust platforms with our enterprise services for robotics companies and provide a production grade solution to your customer.</h4>
+    <p>Our services allow robotics companies to reduce overhead costs and operational expenses for deployment, maintenance, security, and support. Our portfolio covers each stage of your go-to market, while helping you spend more time on your innovation. Our services for robotics companies signed:</p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:277]:
```suggestion
    <p>Our services allow robotics companies to reduce overhead costs and operational expenses for deployment, maintenance, security, and support. Our portfolio covers each stage of your go-to market, while helping you spend more time on your innovation. Our services for robotics companies include:</p>
```

File: templates/robotics/index.html [hunk 9]
```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
...
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Device Enablement and Maintenance</h3>
+      <h4>Your own production-grade image in our testing infrastructure</h4>
+      <p>Embedded applications depend on the kernel, drivers, libraries, filesystems, and runtime environment. How can you guarantee that your robot will continue working after critical security upgrades to the base kernel and the OS?</p>
+      <p>Our Enablement services allow you to get a production-grade Ubuntu image for your robot, while our Maintenance services guarantee that any security update won't break your device.</p>
+      <p>
+        No regressions, no surprises. 
+      </p>
+      <p><a href="/core/contact-us">Reach out to learn more about enablement and maintenance&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/robotics/index.html:288]: Should this be interactive?

File: templates/robotics/index.html [hunk 10]
```diff
@@ -167,99 +236,97 @@ <h3>System integrity: assured</h3>
...
+      <p><a href="/security/certifications">Learn more about security certification&nbsp;&rsaquo;</a></p>
     </div>
   </div>
+
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip">
   <div class="u-fixed-width">
     <h2>Ready for all boards, frameworks and applications</h2>
```
Comment (anthonydillon) [inline templates/robotics/index.html:331]:
```suggestion
    <h2>Ready for all boards, frameworks, and applications</h2>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments in the code

File: templates/robotics/index.html (hunk 4 above)
Comment (carkod) [reply templates/robotics/index.html:98]: Aren't the ones we are using designed by a designer? The come from assets server

File: templates/robotics/index.html (hunk 2 above)
Comment (carkod) [reply templates/robotics/index.html]: Removed

File: templates/robotics/index.html [hunk 11]
```diff
@@ -271,24 +336,22 @@ <h3>Compatible with popular open source robotics frameworks</h3>
...
     </div>
   </div>
 
-  <div class="u-fixed-width u-sv3">
-    <a href="https://snapcraft.io/blog/your-first-robot-a-beginners-guide-to-ros-and-ubuntu-core-1-5">Build your robot with ROS on Ubuntu Core</a>
-  </div>
-
   <div class="u-fixed-width">
-    <h3>Broad support for development boards</h3>
+    <h3>Enabling robotics boards with our ODMs partners</h3>
```
Comment (minkyngkm) [inline templates/robotics/index.html:354]: "NVIDIA Jetson" seems to be removed in this section according to the copy doc.

File: templates/robotics/index.html [hunk 12]
```diff
@@ -369,12 +432,10 @@ <h3>Dragonboard</h3>
...
   </div>
   <div class="u-fixed-width">
-    <p><a href="https://ubuntu.com/blog/porting-ubuntu-core-18-to-nvidia-jetson-tx1-developer-kit">Read more about building Ubuntu Core for NVIDIA Jetson&nbsp;&rsaquo;</a></p>
+    <p>Check the <a href="/certified/devices">full list of Ubuntu certified hardware</a> and find the right one for your application.</p>
   </div>
 </section>
 
-{% include "robotics/shared/_learn-more-about-robotics.html" %}
-
 <section class="p-strip is-bordered">
```
Comment (minkyngkm) [inline templates/robotics/index.html:439]: This section seems to be removed in the copy doc.

File: templates/robotics/index.html [hunk 13]
```diff
@@ -133,32 +176,56 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
   <div class="u-fixed-width">
-    <p class="p-heading--2">Enhanced security for mission-critical robots</p>
-    <p>Several layers of security are stacked in Ubuntu Core to keep potential surfaces of attack as low as possible for your product deployed in the field.</p>
-    <a href="/engage/securing-ros-on-robotics-platforms-whitepaper">Read the Securing ROS on robotics platforms whitepaper&nbsp;&rsaquo;</a>
+    <h2 class="p-heading--2">Bringing Ubuntu's worldwide infrastructure to your robot</h2>
+    <h4>Deploying a robotics solution? For more than a decade, Ubuntu has provided the best development environment for robotics developers.</h4>
+    <p>Together with <a href="/core">Ubuntu Core</a> — OS optimised for decentralised computing environments — and <a href="https://snapcraft.io/docs/robotics">snap</a> — fully confined embedded containers — developers can easily build and securely deploy their devices. And for the software management of your device, the <a href="/internet-of-things/appstore">IoT Snap Store</a> allows developers to unlock the reliable and widely used update infrastructure of Ubuntu applications in their robots.</p>
+    <div class="u-fixed-width">
```
Comment (minkyngkm) [inline templates/robotics/index.html:188]: This image looks a bit horizontally pressed for me but please disregard it if it looks okay for you! 
image

**Review by minkyngkm — COMMENTED** [review]
Looking good @carkod! I've made some comments for small things. Also, should all the images be hidden on mobile screens?

File: templates/robotics/index.html (hunk 13 above)
Comment (carkod) [reply templates/robotics/index.html:188]: In your screenshot it does look a bit weird, but I don't know how to reproduce that. Maybe, can you hard reload or open the page in incognito? This is what I see:

robot-diagram

**Comment by carkod** [comment]
> Looking good @carkod! I've made some comments for small things. Also, should all the images be hidden on mobile screens?

Yes, as far as I know, the are usually hidden to avoid people scrolling very far? Most of these "decorative" images that I've seen seem to be hidden.

File: templates/robotics/index.html (hunk 13 above)
Comment (minkyngkm) [inline templates/robotics/index.html:188]:
```suggestion
    <div class="u-hide--small">
```
or 

```suggestion
    <div class="u-hide--small u-hide--medium">
```

File: templates/robotics/index.html [hunk 14]
```diff
@@ -133,32 +176,56 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
-        url="https://assets.ubuntu.com/v1/338124d1-shields-security.svg",
+        image (
+        url="https://assets.ubuntu.com/v1/c711b814-reliable_updates.svg",
         alt="",
-        height="200",
-        width="350",
+        width="426",
+        height="250",
         hi_def=True,
+        loading="auto|lazy"
```
Comment (minkyngkm) [inline templates/robotics/index.html:228]:
```suggestion
        loading="lazy"
```

**Review by minkyngkm — APPROVED** [review]
LGTM! just left a few comments for minor things! Thanks Carlos! 👍

**Comment by carkod** [comment]
The demo doesn't seem to be working and there are a few thiings unresolved in the comments, so I think this should be PdM reviewed in a different PR.

**Comment by GAN-Canonical** [comment]
(I was writing my review - but the demo is no longer working) Thanks team for all the work! we are getting there. 

Here is some feedback:

1) Please the ABB and Open Robotics logos are bigger than the other logos:
image

2) The alignment of the text in this section is not great. I know this is standard, but you see that with a diagram it is more noticeable:
image

3) This image is smaller than its counterpart - please can we size it better to cover more area?
image

4)

Also) when you share the link to the page you get this image: 
image
how can we change it?

**Comment by carkod** [comment]
> (I was writing my review - but the demo is no longer working - **Please let me know if I should continue with my feedback here**)
> 
> Thanks team for all the work! we are getting there.
> 
> Here is some feedback:
> 
> 1. Please the ABB and Open Robotics logos are bigger than the other logos:
>    image
> 2. The alignment of the text in this section is not great. I know this is standard, but you see that with a diagram it is more noticeable:
>    image
> 3. This image is smaller than its counterpart - please can we size it better to cover more area?
>    image
> 
> Also) when you share the link to the page you get this image: image how can we change it?

4. What do you want to change it to? @sowasred2012 do we need a designer to change this image?

**Comment by GAN-Canonical** [comment]
@carkod this can work. Maybe @waynecrosby could have a look (is in [canva again](https://www.canva.com/design/DAFBP5-IsNE/8FRH7SXndJ_IoP5OehxvmQ/edit?utm_content=DAFBP5-IsNE&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton))
Robotics webpage

**Comment by GAN-Canonical** [comment]
@carkod the current diagram doesn't work for mobile, so please, could we use this instead? 
Right-OS-Diagram-Mobile

**Comment by carkod** [comment]
> @carkod this can work. Maybe @waynecrosby could have a look (is in [canva again](https://www.canva.com/design/DAFBP5-IsNE/8FRH7SXndJ_IoP5OehxvmQ/edit?utm_content=DAFBP5-IsNE&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)) Robotics webpage

I don't think we can use this image, it has the new Ubuntu logo which is still hasn't been unveiled...

**Comment by GAN-Canonical** [comment]
> > @carkod this can work. Maybe @waynecrosby could have a look (is in [canva again](https://www.canva.com/design/DAFBP5-IsNE/8FRH7SXndJ_IoP5OehxvmQ/edit?utm_content=DAFBP5-IsNE&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton)) Robotics webpage
> 
> I don't think we can use this image, it has the new Ubuntu logo which still hasn't been unveiled...

I think we can - we unveiled it at kubecon. @waynecrosby - sorry, please, could you confirm?

**Comment by GAN-Canonical** [comment]
@sowasred2012 @carkod please could you enable the demo again?

**Comment by carkod** [comment]
> @carkod the current diagram doesn't work for mobile, so please, could we use this instead on mobile only?

@GAN-Canonical do you have an asset link? or the original link? It doesn't look right on github 
https://user-images.githubusercontent.com/78618068/169572316-3fc373a1-6196-442a-b4d1-1a31b2f8358d.svg

**Comment by sowasred2012** [comment]
@carkod when I refresh the page to see the other images, I always see a flash of the initial image (the plough):

Peek 2022-05-23 12-31

I would suggest adding a `u-hide` class to the initial image, and then in your JS, removing that class when the image src has been set. 

Also, I think the images are still too small, maybe @GAN-Canonical would like something more different still, but this is with a 5/7 split on the columns.

Peek 2022-05-23 12-32

**Comment by GAN-Canonical** [comment]
> > @carkod the current diagram doesn't work for mobile, so please, could we use this instead on mobile only?
> 
> @GAN-Canonical do you have an asset link? or the original link? It doesn't look right on github https://user-images.githubusercontent.com/78618068/169572316-3fc373a1-6196-442a-b4d1-1a31b2f8358d.svg
> 
> Also the demo should be working now.
> 
> Pending items:
> 
> * Social image to be checked by @waynecrosby
> * Asset link or original link of the mobile view diagram
> * @GAN-Canonical to confirm we want the latest copy updates added.
> * Missing fourth image in the header "image randomizer"

@carkod here is the image https://assets.ubuntu.com/v1/c90df7a5-Right-OS-Diagram-Mobile.svg 

Please, as I review the demo - I might do some copy updates. I will ping you when done. 

I also wanted the header image bigger - @sowasred2012 's 5/7 split was attractive - please could you implement that one on the demo for review?

**Comment by carkod** [comment]
> @carkod when I refresh the page to see the other images, I always see a flash of the initial image (the plough):
> 
> Peek 2022-05-23 12-31 [ ](https://user-images.githubusercontent.com/2376968/169810437-fac5c7b2-52de-4fda-a45c-e686a2a88799.gif) Peek 2022-05-23 12-31 [ ](https://user-images.githubusercontent.com/2376968/169810437-fac5c7b2-52de-4fda-a45c-e686a2a88799.gif)
> 
> I would suggest adding a `u-hide` class to the initial image, and then in your JS, removing that class when the image src has been set.
> 
> Also, I think the images are still too small, maybe @GAN-Canonical would like something more different still, but this is with a 5/7 split on the columns.
> 
>

I can do that.
But the problem I see with this approach is that the image is so big, it's pushing the container, so you will see another "plough" or page jump.

**Comment by GAN-Canonical** [comment]
@carkod please, go ahead with the copy doc changes. Having the demo makes a whole difference as I can finally see how it looks on the web.  
Thank you for your help.

**Comment by waynecrosby** [comment]
@carkod @GAN-Canonical  We can't use this logo unfortunately as it has not been released yet, we used a different one at Kubecon. We could just use the new Ubuntu logo without Canonical. I think we should stick with the old Suru theme too as that has not been officially released either. I'll take a look in Canva and advise, Thanks

**Comment by sowasred2012** [comment]
@GAN-Canonical @waynecrosby were there any updates on this?

**Comment by GAN-Canonical** [comment]
@carkod Please, on the copy doc there are 5 new small copy doc changes (1 with the link to all the new images for the header & and the rest capitalizing some letters) 

Also - this is the new thumbnail for the webpage  

Robotics webpage (2)

**Comment by davegoddard42** [comment]
LGTM @sowasred2012 

We can iterate on the mobile design in the coming iterations.

**Comment by GAN-Canonical** [comment]
@sowasred2012 the interactive diagram looks really good! I like it!
There are a few copy doc changes - please could you have a look before going live?

File: templates/robotics/index.html [hunk 15]
```diff
@@ -2,42 +2,59 @@
...
 </section>
 
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Ubuntu is all your robot needs</h2>
+    <p class="p-heading--4">We provide you with the essential infrastructure to easily, securely, and reliably deploy your robots</p>
+    <p>Our open source tools help you:</p>
+    <ul class="p-list">
+      <li class="p-list__item is-ticked">Keep your robots up-to-date efficiently and cost-effectively with over-the-air and low bandwidth updates with failure rollback.</li>
+      <li class="p-list__item is-ticked">Comply with cybersecurity requirements with build in security tools and security maintenance.</li>
```
Comment (minkyngkm) [inline templates/robotics/index.html:43]:
```suggestion
      <li class="p-list__item is-ticked">Comply with cybersecurity requirements with built-in security tools and security maintenance.</li>
```

File: templates/robotics/index.html [hunk 16]
```diff
@@ -131,169 +149,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
+      </div>
+
+      <div class="u-hide--large js-robotics-content" id="robotics-content-2">
+        <h3 class="p-heading--2">Ubuntu</h3>
+        <p>Ubuntu is an open-source operating system. For more than a decade, Ubuntu has supported key robotics open-source projects, such as <a href="/robotics/what-is-ros">ROS</a>, PX4, Autoware, Open-CV, PCL, and more.</p>
+        <p>Every aspect of the development process benefits from Ubuntu's responsiveness, ease of use, regular software updates, lightweight nature, and a high degree of security.</p>
+
+        <p>
+          <a href="https://www.ubuntu.com/" class="p-button--positive">Learn more about Ubuntu</a>
```
Comment (minkyngkm) [inline templates/robotics/index.html:235]:
```suggestion
          <a href="/" class="p-button--positive">Learn more about Ubuntu</a>
```

File: templates/robotics/index.html [hunk 17]
```diff
@@ -131,169 +149,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
+<section class="p-strip">
+  {% include "robotics/shared/_learn-more-about-robotics.html" %}
+</section>
 
+<section class="p-strip--light">
   <div class="u-fixed-width">
-    <h3>Compatible with popular open source robotics frameworks</h3>
+    <h2>Enterprise services for robotics companies</h2>
+    <p class="p-heading--4">Deploy robust platforms with our enterprise services for robotics companies and provide a production grade solution to your customer.</p>
+    <p>Our services allow robotics companies to reduce overhead costs and operational expenses for deployment, maintenance, security, and support. Our portfolio covers each stage of your go-to market, while helping you spend more time on your innovation. Our services for robotics companies include:</p>
```
Comment (minkyngkm) [inline templates/robotics/index.html:341]:
```suggestion
    <p>Our services allow robotics companies to reduce overhead costs and operational expenses for deployment, maintenance, security, and support. Our services for robotics companies include:</p>
```

File: templates/robotics/index.html [hunk 18]
```diff
@@ -131,169 +139,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
-      <p class="p-card__desc">ROS modularity is best leveraged in the Ubuntu environment. Snapping ROS modules helps manage the complexity of ROS based applications.</p>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Device Enablement and Maintenance</h3>
+      <h4>Your own production-grade image in our testing infrastructure</h4>
+      <p>Embedded applications depend on the kernel, drivers, libraries, file systems, and runtime environment. How can you guarantee that your robot will continue working after critical security upgrades to the base kernel and the OS?</p>
+      <p>Our Enablement services allow you to get a production-grade Ubuntu image for your robot, while our Maintenance services guarantee that any security update won't break your device.</p>
+      <p>
+        No regressions, no surprises. 
+      </p>
+      <a class="p-button--positive js-invoke-modal" href="/core/contact-us"> Reach out to hear more&nbsp;&rsaquo;</a>
```
Comment (minkyngkm) [inline templates/robotics/index.html:342]:
```suggestion
      <a class="p-button--positive js-invoke-modal" href="/core/contact-us"> Reach out to hear more</a>
```

File: templates/robotics/index.html [hunk 19]
```diff
@@ -131,169 +139,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
-      <p class="p-card__desc">Deep learning machine vision applications can be developed with OpenCV on top of Ubuntu.</p>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">ROS Enterprise Support and Security Maintenance</h3>
+      <h4>Hardened ROS with 10-year security in partnership with Open Robotics</h4>
+      <p><a href="/robotics/ros-esm">ROS ESM</a> is a service that provides security maintenance for your ROS distribution, EOL, and non-EOL, providing you with security fixes for your core ROS dependencies.</p>
+      <p>
+        In partnership with <a href="https://www.openrobotics.org/">Open Robotics</a>, maintainer of ROS, ROS ESM also includes ROS enterprise support: a 24/7 support centre to request fixes to core ROS libraries.
+      </p>
+      <p>No more neglecting security updates or waiting for fixes. </p>
+      <p><a class="p-button--positive" href="/robotics/ros-esm">Learn about ROS ESM&nbsp;&rsaquo;</a></p>
```
Comment (minkyngkm) [inline templates/robotics/index.html:352]:
```suggestion
      <p><a class="p-button--positive" href="/robotics/ros-esm">Learn about ROS ESM;</a></p>
```

File: templates/robotics/index.html [hunk 20]
```diff
@@ -131,169 +139,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
+      <p class="p-card__content">Treat every device on your fleet as a first-class server-grade managed asset, with monitoring, security, role-based access controls, and application lifecycle management. The embedding program includes:</p>
+      <ul>
+        <li>Security maintenance for Ubuntu Universe and Main</li>
+        <li>Ubuntu systems management with <a href="https://landscape.canonical.com">Landscape</a></li>
+        <li><a href="/security/livepatch/">Kernel Livepatch</a> service to avoid reboots</li>
+        <li><a href="/security/certifications">Security certification</a></li>
+        <li>24/7, open source <a href="/support">software support</a> for the full stack.</li>
+        <li>Trademark use</li>
+      </ul>
+      <p><a class="p-button--positive" href="/embedding">Learn about Embedding&nbsp;&rsaquo;</a></p>
```
Comment (minkyngkm) [inline templates/robotics/index.html:369]:
```suggestion
      <p><a class="p-button--positive" href="/embedding">Learn about Embedding</a></p>
```

File: templates/robotics/index.html [hunk 21]
```diff
@@ -131,169 +139,260 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
...
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Robotics Security Compliance</h3>
+      <h4>Security updates will become your ally</h4>
+      <p>Whatever cybersecurity framework you have chosen, including ISO 27000, NIST, PCI, or CIS Controls, Ubuntu helps you stay compliant and reduces your operational risk.</p>
+      <p>
+        Access automation for hardening and compliance profiles, such as CIS and DISA-STIG, as well as the FIPS 140-2 and Common Criteria certifications.
+      </p>
+      <p><a class="p-button--positive" href="/security/certifications">Learn about security certification&nbsp;&rsaquo;</a></p>
```
Comment (minkyngkm) [inline templates/robotics/index.html:378]:
```suggestion
      <p><a class="p-button--positive" href="/security/certifications">Learn about security certification</a></p>
```

**Review by minkyngkm — APPROVED** [review]
Would it be better to take out `>` from buttons?
image

Other than that, LGTM! Nice work Scott! 🏅

**Pull request merged**